client/app/utils/i18n.js

```diff
@@ -13,35 +13,33 @@
 
 import { bind } from 'lodash';
 
-function isServer() {
-  return typeof window === 'undefined';
-}
-
-if (isServer()) {
-
-  // Initialize global.I18n
-  // In browser we initialize this in a script-tag manually
-  global.I18n = {}; // eslint-disable-line no-undef
-
-  // Load the translation bundle in the global.I18n variable.
-  // In browser the bundle is loaded in a separate script-tag.
-
-  try {
-    // The translation bundle will be loaded to the global I18n
-    // variable. Initialize the variable here.
-    require('../i18n/all.js'); // eslint-disable-line no-undef
-  } catch (e) {
-    console.warn("Can't load language bundle all.js"); // eslint-disable-line no-console
-  }
-}
+// Initialize global.I18n
+// This might be initialized already, because in browser
+// we do initialize this before the translations get loaded
+// in a separate script tag
+global.I18n = global.I18n || {}; // eslint-disable-line no-undef
```

Comment:
Now that this is in the top level module scope, what does `global` mean in the browser's context?